package.json

```diff
@@ -28,7 +28,7 @@
   "activationEvents": [
     "onLanguage:polarity"
   ],
-  "main": "./out/extension.js",
+  "main": "./dist/extension",
```

Comment:
Why is this missing the `.js` extension?

Reply:
I think it accepts both: with and without the `.js` extension. In the guide they use it without the extension in this paragraph: https://code.visualstudio.com/api/working-with-extensions/bundling-extension#run-the-extension